### PR TITLE
Improve `./start` to work on Windows

### PR DESCRIPTION
**Pull request by Eric-Arellano**
Closes https://github.com/Qiskit/documentation/issues/1607. There are two issues:

1. The shebang will be ignored, so you need to run `python ./start` instead of `./start`
2. Volume mounting with Docker on Windows uses Unix-style path, per https://medium.com/@kale.miller96/how-to-mount-your-current-working-directory-to-your-docker-container-in-windows-74e47fa104d7

### CODE REVIEW

**Comment by Eric-Arellano** [comment]
TODO: confirm with @beckykd this fixes things.

**Comment by Eric-Arellano** [comment]
@frankharkins @arnaucasau before landing this, please see if the status quo works with Git Bash for Becky. If it does, please confirm this change does not break Git Bash and it also fixes PowerShell.

Imo supporting PowerShell is not essential, so long as Git Bash works on Windows.

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ You can preview the docs locally by following these two steps:
 
 1. Ensure Docker is running. For example, open Rancher Desktop.
 2. Run `./start` in your terminal, and open http://localhost:3000 in your browser.
+   - On Windows, run `python start` instead.
```
Comment (Eric-Arellano) [inline README.md:80]:
```suggestion
   - On Windows, run `python start` instead. Alternatively, use Windows Subsystem for Linux and run `./start`.
```

**Comment by Eric-Arellano** [comment]
We got it working on both Powershell and Git Bash 🚀

**Comment by Eric-Arellano** [comment]
@arnaucasau @frankharkins bump on reviewing this please

**Review by arnaucasau — APPROVED** [review]
Awesome! I tested the script on a Windows machine using PowerShell and Git Bash and worked well! Thanks :rocket:

**Pull request merged**